code/workspaces/web-app/src/selectors/routerSelectors.js

```diff
@@ -1,4 +1,4 @@
-const selectUrlHash = ({ router: { location: { hash } } }) => hash;
+const selectUrlHash = ({ router: { location: { search } } }) => search;
```

Comment:
If this should be the search not the hash, we should rename to selectUrlSearch and follow through the code, or I think it will be confusing.